src/cloudflare/internal/test/to-markdown/to-markdown-api-test.wd-test

```diff
@@ -0,0 +1,36 @@
+using Workerd = import "/workerd/workerd.capnp";
+
+const unitTests :Workerd.Config = (
+  services = [
+    ( name = "to-markdown-api-test",
+      worker = (
+        modules = [
+          (name = "worker", esModule = embed "to-markdown-api-test.js")
+        ],
+        compatibilityDate = "2023-01-15",
```

Comment:
is  this intentional?

Reply:
I copied some tests and modified them to exercise the new service itself. Didn't pay much attention to compat dates